yarn.lock

```diff
@@ -504,7 +512,7 @@ core-util-is@~1.0.0:
 
 "cql-execution@https://github.com/cqframework/cql-execution#master":
   version "1.3.4"
-  resolved "https://github.com/cqframework/cql-execution#4f9535a7fbcff5923b11ddf7dafe55442c81d1fa"
+  resolved "https://github.com/cqframework/cql-execution#b30335163482b982dfc15dc504a7340a7161b790"
```

Comment:
It seems there have been 2 more commits to cql-execution master branch over the past day or 2 so this may need to be updated.

Reply:
Good catch--there will be a separate PR to point to the published cql-execution npm package and update the README--so this is OK for this PR